curriculum/challenges/english/02-javascript-algorithms-and-data-structures/functional-programming/avoid-mutations-and-side-effects-using-functional-programming.english.md

```diff
@@ -30,6 +30,13 @@ tests:
     testString: incrementer(); assert(fixedValue === 4);
   - text: Your <code>incrementer</code> function should return a value that is one larger than the <code>fixedValue</code> value.
     testString: const newValue = incrementer(); assert(newValue === 5);
```

Comment:
Kind of unrelated to this change, but noticed several people on the forum were creating a variable outside the function named `newValue` for testing purposes which caused correct code to fail on this test.  It would be highly unlikely someone would choose a name like `__newValue`, so let's change it here. 
```suggestion
    testString: const __newValue = incrementer(); assert(__newValue === 5);
```
This is not necessary for the new test you are adding, because `newValue` is contained in the IIFE.

Reply:
@RandellDawson Hi, Please have a look at the new changes. I am not able to understand the build process, why exactly the build failed.